tests/test_core/test_alignment.py

```diff
@@ -244,6 +244,64 @@ def setUp(self):
         self.b = Alignment(["AAA", "AAA"])
         self.c = SequenceCollection(["AAA", "AAA"])
 
+    def test_deepcopy(self):
+        """correctly deep copy aligned objects in an alignment"""
```

Comment:
These tests are incorrect. The property of deepcopy means the new object at all layers must be different. 

1 - you need to use `self.Class` to create the object. Since the test suite has different subclasses, all of them will be executed
2 - skip annotation checking since that code is tested elsewhere
3 - you need to check the `sliced` argument does not raise exceptions
4 - add a `AlignmentTests.test_sliced_deepcopy` method. In that case, you should create a bigger alignment and slice it. Then compare the results of `deepcopy(sliced=False)` with `deepcopy(sliced=True)` the length of underlying sequences should differ.